inout.py

```diff
@@ -63,6 +63,20 @@ def fileDictHeaderLists(file, key, val, delim="\t"):
                 fileD[cols[headD[key]]].append(cols[headD[val]])
     return fileD
 
+def fileDictLists(file, delim="\t"):
+    fileD=defaultdict(list)
+    with open(file, "r") as fin:
+        for line in fin:
+            rows = line.rstrip("\n").split(delim)
+            i=0
```

Comment:
After implementing the change above, you won't actually need the i counter anymore. For each row, you will just be appending the value in the 'val' column to the list associated with the key contained in the 'key' column.